Cogs/CogManagement.py

```diff
@@ -13,20 +13,54 @@ def __init__(self, bot):
     @commands.command()
     @commands.has_permissions(administrator=True)
     async def reload(self, ctx, cog_name):
+        """Reload a specific cog
+        Take in the name of single cog from a user and reload it. Output a message confirming reload action
+        and use reload extension method on the cog. If reloading server managment, call load_server_managment
+        method from CogManagment.py
+
+        Args:
+            cog_name: Name of the cog that will be reloaded
```

Comment:
`cog_name` in all the functions in the file needs a type

Reply:
Resolved by d6472d7